test/hooks.tsx

```diff
@@ -1,216 +1,227 @@
-import {serial as test} from 'ava';
-import {spawn} from 'node-pty';
-import stripAnsi from 'strip-ansi';
-
-const term = (fixture: string, args: string[] = []) => {
-	let resolve: (value?: any) => void;
-	let reject: (error?: Error) => void;
-
-	// eslint-disable-next-line promise/param-names
-	const exitPromise = new Promise((resolve2, reject2) => {
-		resolve = resolve2;
-		reject = reject2;
-	});
-
-	const env = {...process.env};
-	delete env.CI;
-
-	const ps = spawn('ts-node', [`./fixtures/${fixture}.tsx`, ...args], {
-		name: 'xterm-color',
-		cols: 100,
-		cwd: __dirname,
-		env
-	});
-
-	const result = {
-		write: (input: string) => {
-			// Give TS and Ink time to start up and render UI
-			// TODO: Send a signal from the Ink process when it's ready to accept input instead
-			setTimeout(() => {
-				ps.write(input);
-			}, 3000);
-		},
-		output: '',
-		waitForExit: () => exitPromise
-	};
-
-	ps.on('data', data => {
-		result.output += data;
-	});
-
-	ps.on('exit', code => {
-		if (code === 0) {
-			resolve();
-			return;
-		}
-
-		reject(new Error(`Process exited with non-zero exit code: ${code}`));
-	});
-
-	return result;
-};
-
-test('useInput - handle lowercase character', async t => {
-	const ps = term('use-input', ['lowercase']);
-	ps.write('q');
-	await ps.waitForExit();
```

Comment:
@sindresorhus / @vadimdemedes I'm having issues with `node-pty`. When this line runs, the Node process blows up with the following stacktrace:

```bash
FATAL ERROR: HandleScope::HandleScope Entering the V8 API without proper locking in place
 1: 0x100eda200 node::Abort() [/Users/pepicrft/.asdf/installs/nodejs/18.12.1/bin/node]
 2: 0x100eda2f8 node::OOMErrorHandler(char const*, bool) [/Users/pepicrft/.asdf/installs/nodejs/18.12.1/bin/node]
 3: 0x101022aa8 v8::HandleScope::HandleScope(v8::Isolate*) [/Users/pepicrft/.asdf/installs/nodejs/18.12.1/bin/node]
```

I did some searching but I couldn't find anything useful to fix the issue. Does anything come to your mind regarding what might be causing this?